src/middlewared/middlewared/service/core_service.py

```diff
@@ -887,7 +887,7 @@ async def set_options(self, app, options):
         if "py_exceptions" in options:
             app.py_exceptions = options["py_exceptions"]
 
-    @api_method(CoreSubscribeArgs, CoreSubscribeResult)
+    @api_method(CoreSubscribeArgs, CoreSubscribeResult, authentication_required=False)
```

Comment:
We just removed necessity of having `authentication_required` on these endpoints in 481ad9c6b4701bac21c28ce16ad33a94b414bb1e  but this is adding it back.

Change this to `authorization_required=False`

Reply:
Nvm, this was resolved in https://github.com/truenas/middleware/pull/15352. Just need to rebase your PR against latest master.